crates/sqlbuiltins/src/functions/aggregates.rs

```diff
@@ -0,0 +1,363 @@
+    description => "Gives the approximate percentile of a column"
+);
+
+make_const!(
+    APPROX_PERCENTILE_CONT_WITH_WEIGHT,
+    name => "approx_percentile_cont_with_weight",
+    example => "approx_percentile_cont_with_weight(a)",
+    description => "Gives the approximate percentile of a column with a weight column"
+);
+
+make_const!(
+    ARRAY_AGG,
+    name => "array_agg",
+    example => "array_agg(a)",
+    description => "Returns a list containing all the values of a column"
+);
+make_const!(
+    AVG,
+    name => "avg",
+    example => "avg(a)",
+    description => "Returns the average of a column"
+);
+make_const!(
+    BIT_AND,
+    name => "bit_and",
+    example => "bit_and(a)",
+    description => "Returns the bitwise AND of a column"
+);
+make_const!(
+    BIT_OR,
+    name => "bit_or",
+    example => "bit_or(a)",
+    description => "Returns the bitwise OR of a column"
+);
+make_const!(
+    BIT_XOR,
+    name => "bit_xor",
+    example => "bit_xor(a)",
+    description => "Returns the bitwise XOR of a column"
+);
+make_const!(
+    BOOL_AND,
+    name => "bool_and",
+    example => "bool_and(a)",
+    description => "Returns the boolean AND of a column"
+);
+make_const!(
+    BOOL_OR,
+    name => "bool_or",
+    example => "bool_or(a)",
+    description => "Returns the boolean OR of a column"
+);
+make_const!(
+    CORRELATION,
+    name => "correlation",
+    example => "correlation(x, y)",
+    description => "Returns the correlation coefficient of two columns"
+);
+make_const!(
+    COUNT,
+    name => "count",
+    example => "count(a)",
+    description => "Returns the number of rows in a column"
+);
+make_const!(
+    COVARIANCE,
+    name => "covariance",
+    example => "covariance(x, y)",
+    description => "Returns the covariance of two columns"
+);
+make_const!(
+    COVARIANCE_POP,
+    name => "covariance_pop",
+    example => "covariance_pop(x, y)",
+    description => "Returns the population covariance of two columns"
+);
+make_const!(
+    FIRST_VALUE,
+    name => "first_value",
+    example => "first_value(a)",
+    description => "Returns the first value in a column"
+);
+make_const!(
+    GROUPING,
+    name => "grouping",
+    example => "grouping(a)",
+    description => "Returns 1 if a column is aggregated, 0 otherwise"
+);
+make_const!(
+    LAST_VALUE,
+    name => "last_value",
+    example => "last_value(a)",
+    description => "Returns the last value in a column"
+);
+make_const!(
+    MAX,
+    name => "max",
+    example => "max(a)",
+    description => "Returns the maximum value in a column"
+);
+make_const!(
+    MEDIAN,
+    name => "median",
+    example => "median(a)",
+    description => "Returns the median value in a column"
+);
+make_const!(
+    MIN,
+    name => "min",
+    example => "min(a)",
+    description => "Returns the minimum value in a column"
+);
+make_const!(
+    REGR_AVGX,
+    name => "regr_avgx",
+    example => "regr_avgx(y, x)",
```

Comment:
Yes, typically things are ordered alphabetically like x, y, but in statistical functions like `regr_avgx`, the order `y, x` is used because it follows the convention of modeling `y` (dependent variable) as a function of `x` (independent variable). This is standard in statistical and regression analysis contexts, despite the potential for initial mental friction. 

It's also consistent with other doc examples.

https://docs.snowflake.com/en/sql-reference/functions/regr_avgx
https://duckdb.org/docs/sql/aggregates.html